src/flatten_to_stream.rs

```diff
@@ -0,0 +1,101 @@
+use core::mem;
+use core::marker;
+use {Async, Future, Poll};
+use stream::Stream;
+
+/// Future for the `flatten_to_stream` combinator, flattening a future-of-a-stream to get just
+/// the result of the final future as a stream.
+///
+/// This is created by the `Future::flatten_to_stream` method.
+#[must_use = "streams do nothing unless polled"]
+pub struct FlattenToStream<F>
+    where
+        F : Future,
```

Comment:
Typically we put this on the same line as `where` above, and omit the space before `:`